scripts/install-python.sh

```diff
@@ -7,8 +7,14 @@
 # Installs python 3.10; assumes GitHub's ubuntu-latest
 
 echo "Installing python 3.10"
+sudo add-apt-repository -q ppa:deadsnakes/ppa
 sudo apt-get install -qq python3.10 > /dev/null
 sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.10 1
 sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.10 1
 python3 --version
-pip3 install -q --upgrade pip setuptools==70.0.0 launchpadlib wheel
+
+# setuptools and wheel versions must be matched to working releases
+# we need an old version of setuptools (#381) and newer versions of
```

Comment:
the `#381` is a bit ambiguous

Reply:
It's referring to https://github.com/seL4/ci-actions/pull/381 which is what I would expect from an unqualified issue number, so I think this one is fine.

Reply:
Okay no worries then.